src/quo/components/selectors/disclaimer/view.cljs

```diff
@@ -3,21 +3,23 @@
     [quo.components.markdown.text :as text]
     [quo.components.selectors.disclaimer.style :as style]
     [quo.components.selectors.selectors.view :as selectors]
+    [quo.theme]
     [react-native.core :as rn]))
 
 (defn view
   [{:keys [checked? blur? accessibility-label container-style on-change]} label]
-  [rn/touchable-without-feedback
-   {:on-press            on-change
-    :accessibility-label :disclaimer-touchable-opacity}
-   [rn/view {:style (merge container-style (style/container blur?))}
-    [selectors/view
-     {:type                :checkbox
-      :accessibility-label accessibility-label
-      :blur?               blur?
-      :checked?            checked?
-      :on-change           on-change}]
-    [text/text
-     {:size  :paragraph-2
-      :style style/text}
-     label]]])
+  (let [theme (quo.theme/use-theme)]
```

Comment:
You might need `use-theme-value` here, because `use-theme` returns map `{:theme "light"}`

Reply:
good catch, i knew that, never do something for the future, which makes your life harder in present

Reply:
Can we instead change it so `use-theme` returns the value instead of the map? It doesn't seem to be used anywhere else, other than internally or by mistake

Reply:
Originally the intention was to use a map to support more themeable data so to speak, but this never happened until now, so it makes sense to return the value directly. @J-Son89 probably knows better the original intention

Reply:
Yep that was it. Fine to change from my side 👍

Reply:
@J-Son89 @ilmotta Then I'll deprecate `use-theme-value` and will add a `use-theme` that does the same thing. Will rename the original `use-theme` to smth else.

Reply:
Created a follow-up [issue](https://github.com/status-im/status-mobile/issues/19602)